vcr/assets/NutsAuthorizationCredential.config.yaml

```diff
@@ -0,0 +1,18 @@
+concept: authorization
+credentialType: NutsAuthorizationCredential
+indices:
+  - name: index_id
+    parts:
+      - path: id
+  - name: index_issuer
+    parts:
+      - path: issuer
+  - name: index_holder
+    parts:
+      - path: credentialSubject.id
+      - path: credentialSubject.purposeOfUse
+      - path: issuanceDate
```

Comment:
why is `issuanceDate` part of this index?

Second though: unless we'll be searching for an exact value, using the normal index type for dates isn't really the best solution? (probably better to convert it to a number and index that, for range queries and such?)

Reply:
It's currently not possible to use optional fields for an index (with nil and such, it would create large index entries). An index on only holder and purpose would not be specific enough. The extra index on date would help in finding eTransfer authorization credentials from the last 4 weeks or so. the issuanceDate field is specced as RFC3339 date which when translated from string to bytes is comparable.

Reply:
Doing a range query over an RFC3339 string date is a pain though, but I guess good enough for now. Future feature.